### ISSUE TITLE
Secure headers incorrectly suppress OnAttributeChanged handler

### ISSUE THREAD

**Issue by Meorawr** [Stanzilla/WoWUIBugs#219]
As of patch 9.2, secure group and aura headers have developed an issue whereby attributes controlled by the headers can be modified without the OnAttributeChanged script handler from being dispatched as user code is typically expecting.

This was broken by the addition of the third optional boolean parameter to the `Frame:SetAttribute(name, value[, suppressScriptHandlers])` widget API. In the SecureGroupHeaders script there's an `setAttributesWithoutResponse` function wherein [a variable number of returns from a `select` call are passed into a SetAttribute call](https://github.com/Gethe/wow-ui-source/blob/978a1677c7825f6ddbf6a3f276ed61e61c6c970a/Interface/FrameXML/SecureGroupHeaders.lua#L105). As this function is called with pairs of attribute names and values, this leads to the suppression boolean evaluating to true for all SetAttribute calls except for the last one.

tldr; we need this change applied to un-break addons using secure header templates in 9.2:

```diff
diff --git a/Interface/FrameXML/SecureGroupHeaders.lua b/Interface/FrameXML/SecureGroupHeaders.lua
index 0f0aa4dd..71074d61 100644
--- a/Interface/FrameXML/SecureGroupHeaders.lua
+++ b/Interface/FrameXML/SecureGroupHeaders.lua
@@ -102,7 +102,8 @@ local function setAttributesWithoutResponse(self, ...)
 	local oldIgnore = self:GetAttribute("_ignore");
 	self:SetAttribute("_ignore", "attributeChanges");
 	for i = 1, select('#', ...), 2 do
-		self:SetAttribute(select(i, ...));
+		local attr, value = select(i, ...);
+		self:SetAttribute(attr, value);
 	end
 	self:SetAttribute("_ignore", oldIgnore);
 end
```

**Comment by Meorawr** [comment]
Fixed in 9.2.0.42423.
